core/modules/filters/interleave.js

```diff
@@ -0,0 +1,75 @@
+/*\
+title: $:/core/modules/filters/interleave.js
+type: application/javascript
+module-type: filteroperator
+
+Interleave two or more lists one item at a time
+
+\*/
+(function(){
+
+	/*jslint node: true, browser: true */
+	/*global $tw: false */
+	"use strict";
+
+	/*
+	Interleave two or more lists
+	*/
+	exports.interleave = function(source,operator,options) {
+		var allowDuplicates = false;
+		switch(operator.suffix) {
+			case "raw":
+				allowDuplicates = true;
+				break;
+			case "dedupe":
+				allowDuplicates = false;
+				break;
+		}
```

Comment:
I copied that logic block verbatim from the `enlist` operator, figuring that I might want to add other suffixes later and this would be a good place to put them. For example, I'm thinking of adding a "filter mode" to this, like `interleave:filter` that treats its arguments not as spelled-out lists to evaluate, but as subfilters whose results should be interleaved. If I do add "filter mode", it'll probably be in a separate PR.

Reply:
ok. I can live with that.